### PR TITLE
Feature/cinder fixes

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [X] Bug fix
- [ ] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Simple fix to related to volume provisioning with `cinder` as default backend.
### Changes made:

1) Don't request a PVC for the inbox volume if `ingest.coLocate=true`
2) StatefulSets can handle their own volume requests in most cases.

### CODE REVIEW

**Comment by jbygdell** [comment]
Fixing Travis requires changes to the e2e-tester so that is coming as a separate PR.

File: ega-charts/localega/templates/lega-pvc.yaml [hunk 1]
```diff
@@ -15,6 +15,8 @@ spec:
 {{ end }}
 ---
 {{ if and  .Values.persistence.enabled ( not .Values.postgres.persistence.existingClaim ) }}
+{{ if .Values.persistence.storageClass }}
+{{ if or ( eq "local-storage" .Values.persistence.storageClass ) }}
```
Comment (jonandernovella) [inline ega-charts/localega/templates/lega-pvc.yaml]: Are these two conditionals necessary? If so, why? The second if statement will always go through once the first one has passed.

Comment (jonandernovella) [reply ega-charts/localega/templates/lega-pvc.yaml]: Perhaps what you want is an `if and`?

Comment (jbygdell) [reply ega-charts/localega/templates/lega-pvc.yaml]: Nope, forgot to remove the `or`

**Review by blankdots — APPROVED** [review]

**Review by viklund — APPROVED** [review]
LGTM

**Pull request merged**